src/remoteConfig.ts

```diff
@@ -43,7 +42,10 @@ class RemoteConfig {
       throw new Error('remote config not initialized');
     }
 
-    const result = this.gb.getFeatureValue(this.configKey, null);
+    const result = this.gb.getFeatureValue(
+      process.env.API_CONFIG_FEATURE_KEY,
```

Comment:
Add a check like with `GROWTHBOOK_API_CONFIG_CLIENT_KEY` env?

Reply:
no need, it will just be null

Reply:
👍